app/src/main/java/com/babylon/wallet/android/presentation/status/assets/AssetDialogViewModel.kt

```diff
@@ -131,7 +131,7 @@ class AssetDialogViewModel @Inject constructor(
         val accountContext: Network.Account? = null,
     ) : UiState {
 
-        val isLoadingBalance = assetPrice == null
+        val isLoadingBalance = args.underAccountAddress != null && assetPrice == null
```

Comment:
indeed it works, but what I don't understand is how does it work?
What does `underAccountAddress` mean? 

Our goal was to not show the price and shimmering in TX screen, so does this boolean controls this part?

Reply:
let's also wait for @micbakos-rdx  to confirm this fix since he knows this place very well.

Reply:
looking at view model logic and args, we always seem to pass amount via args, and having account address context mean that we are accessing details from a place where we have access to the account. That was my thinking after I looked where we pass `underAccountAddress` arg

Reply:
thanks. Maybe add a comment there!

Reply:
Yeap Jakub is exactly right